contrib/systemd/btrbk.timer.in

```diff
@@ -1,9 +1,10 @@
 [Unit]
 Description=btrbk daily backup
+ConditionPathExists=|@CONFDIR@/btrbk/btrbk.conf
+ConditionPathExists=|@CONFDIR@/btrbk.conf
 
 [Timer]
-OnCalendar=daily
```

Comment:
I think daily is a good default. Some want it hourly, some daily, and I know (at least one) doing it minutely ;-)

Reply:
Forget about that commit... it was late in the night and I was pretty dumb ^^

I completely forgot that the times in the retention policies don't determine how often the tool runs, but how long the snapshots/etc. are kept... so `hourly` would be really bad.

It could however make sense, if somewhere we add the proper systemd way on how to override (described in [systemd.unit(5)](https://manpages.debian.org/testing/systemd/systemd.unit.5.en.html)) the time, namely by adding a file:
`/etc/systemd/system/btrbk.timer.d/foo.conf` with **just** what shall be overridden:
```
[Timer]
OnCalendar=hourly
```